src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -106,12 +106,12 @@ internal partial class Binder
             if (conversion.IsTupleLiteralConversion ||
                 (conversion.IsNullable && conversion.UnderlyingConversions[0].IsTupleLiteralConversion))
             {
-                return CreateTupleLiteralConversion(syntax, (BoundTupleLiteral)source, conversion, isCast: isCast, isExplicitlyNullable: isExplicitlyNullable, destination, diagnostics);
+                return CreateTupleLiteralConversion(syntax, (BoundTupleLiteral)source, conversion, isCast: isCast, conversionGroup, destination, diagnostics);
             }
 
             if (conversion.IsUserDefined)
             {
-                return CreateUserDefinedConversion(syntax, source, conversion, isCast: isCast, isExplicitlyNullable: isExplicitlyNullable, destination, diagnostics);
+                return CreateUserDefinedConversion(syntax, source, conversion, isCast: isCast, conversionGroup ?? new ConversionGroup(conversion), destination, diagnostics);
```

Comment:
>conversionGroup ?? new ConversionGroup(conversion) [](start = 95, length = 50)

Why is this case different? Maybe add a comment.

Reply:
From our discussion, it sounds like the reason we want to attach a conversion group is because there will be an implicit conversion added on top (in the chain of BoundConversion nodes). If that's the case, then we need to make sure that parent has the same conversion group, so it seems it should be created by the parent instead.

---
In reply to: [189102623](https://github.com/dotnet/roslyn/pull/26356#discussion_r189102623) [](ancestors = 189102623)

Reply:
The implicit conversion is added by `CreateUserDefinedConversion`. The reason this caller is different from the other cases in the method is that user-defined conversions will be represented with multiple `BoundConversion` instances so a `ConversionGroup` is needed.

Added a comment.

---
In reply to: [189112933](https://github.com/dotnet/roslyn/pull/26356#discussion_r189112933) [](ancestors = 189112933,189102623)